### PR TITLE
Updated generated files

### PR DESCRIPTION
**Pull request by tomaszslabon**
Updated files that are generated by running the install.sh script.

### CODE REVIEW

File: docs/resources/client-start-help [hunk 1]
```diff
@@ -25,7 +26,7 @@ Flags:
       --tbtc.preParamsGenerationTimeout duration   tECDSA pre-parameters generation timeout. (default 2m0s)
       --tbtc.preParamsGenerationDelay duration     tECDSA pre-parameters generation delay. (default 10s)
       --tbtc.preParamsGenerationConcurrency int    tECDSA pre-parameters generation concurrency. (default 1)
-      --tbtc.keyGenerationConcurrency int          tECDSA key generation concurrency. (default number of cores)
+      --tbtc.keyGenerationConcurrency int          tECDSA key generation concurrency. (default 16)
```
Comment (pdyraga) [inline docs/resources/client-start-help:29]: Let's revert this change, the default should be "number of cores".

There is a problem with a code generating this file as it is generating it based on the current environment.

Comment (tomaszslabon) [reply docs/resources/client-start-help]: Done in 5419dc02e353b706fa483719b947842f58b680f3.

File: pkg/chain/ethereum/ecdsa/gen/abi/WalletRegistry.go [hunk 2]
```diff
@@ -1400,27 +1355,6 @@ func (_WalletRegistry *WalletRegistryTransactorSession) CloseWallet(walletID [32
...
-	return _WalletRegistry.Contract.ForceAddWallet(&_WalletRegistry.TransactOpts, groupPubKey, membersIdsHash)
-}
-
-// ForceAddWallet is a paid mutator transaction binding the contract method 0x2717ee0d.
-//
-// Solidity: function forceAddWallet(bytes groupPubKey, bytes32 membersIdsHash) returns()
-func (_WalletRegistry *WalletRegistryTransactorSession) ForceAddWallet(groupPubKey []byte, membersIdsHash [32]byte) (*types.Transaction, error) {
-	return _WalletRegistry.Contract.ForceAddWallet(&_WalletRegistry.TransactOpts, groupPubKey, membersIdsHash)
-}
-
```
Comment (pdyraga) [inline pkg/chain/ethereum/ecdsa/gen/abi/WalletRegistry.go:1423]: It's really good we cleaned this up, nice job!

**Review by pdyraga — APPROVED** [review]
